### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by GregBreak** [thankarezos/SpeedMalistic#1]
I can load the skin but when i click the rocket nothing happen, all parameters are 0

**Comment by thankarezos** [comment]
Yeah i forgot to mention. You need to open the speedtest.exe file and accept the EULA

**Comment by PaulMcF1987** [comment]
@thankarezos I have the same problem.
I have opened speedtest.exe and accepted the relevant terms. It them runs a speed test in command prompt and closes the box when done.
However, after this, if I click the rocket, it does nothing

**Comment by PaulMcF1987** [comment]
@thankarezos I have the same problem.
I have opened speedtest.exe and accepted the relevant terms. It them runs a speed test in command prompt and closes the box when done.
However, after this, if I click the rocket, it does nothing

**Comment by thankarezos** [comment]
You also need to enable PowerShell scripts, although I am gonna remake the project with a more compact language than PowerShell scripts like Java.
